include/glow/Graph/Graph.h

```diff
@@ -1974,6 +1974,21 @@ class Function final : public IRContainer {
                        float upperFrequency, int64_t filterBankCount,
                        int64_t numCoefficients);
 
+  /// Peformas the ROIAlign operation given the \p featureMap and the \p boxes.
+  /// ROIAlign is similar to crop and resize followed by pooling. The
+  /// co-ordinates to extract the crops are specified in \p boxes. Each cropped
+  /// image has to be resized to have the shape specified by \p outputHeight and
+  /// \p outputWidth. The \p samplingRatio specifies the number of samples to
+  /// take from each bin (along both the dimensions) for the purpose of pooling.
+  /// This node is inspired from onnx
+  /// (https://github.com/onnx/onnx/blob/master/docs/Operators.md#RoiAlign).
+  ROIAlignNode *createROIAlign(llvm::StringRef name, NodeValue featureMap,
+                               NodeValue boxes, NodeValue batchIndices,
+                               std::string mode, uint32_t outputHeight,
+                               uint32_t outputWidth, uint32_t samplingRatio,
+                               float spatialScale, float offset,
+                               bool normalized);
```

Comment:
I see you've added offset and normalized members. These are not part of ONNX format.

Offset: indicates if the sub-resolution center in the pixel is (0,0) or (0.5,0.5) . I know it exists in caffe2 and TF in some ops. Is this the case here? Can this take other values in real world cases?   

Normalized: in loader you set normalized to "false" while the test creates the node with "true".  What sense does this make if the loader always expects the input/output in non-normalized units?

Reply:
I'm trying to land this diff to not delay it anymore. I think offset may be better as a boolean and I'm not sure that "normalized" is required. Let's resolve in follow-ups and fix forward.

Reply:
I added the offset to support both tf(0) and onnx(0.5). However I haven't seen any other values other than 0 and 0.5.  I thought it may help in any future variants, 
The boolean variable normalized is added to support both tf (normalized) and onnx(unnormalized).